lib/vagrant-libvirt/config.rb

```diff
@@ -279,26 +279,12 @@ def _get_cdrom_dev(cdroms)
       end
 
       def _generate_numa
-        raise 'NUMA nodes must be a factor of CPUs' if @cpus % @numa_nodes != 0
+        # todo: compare count of NUMA defined CPUs and throw warning
+        # if config.cpu != NUMA.cpu
```

Comment:
Either add this check now, or make config.cpu the total of NUMA.cpu (and specify in the docs that this is the case).